fourmolu.yaml

```diff
@@ -0,0 +1,8 @@
+indentation: 4
```

Comment:
_BIKESHEDDING ALERT_ I like 2 spaces

Reply:
we're doing 4 spaces normally and 2 space wheres. let it be so.